.changeset/heavy-onions-swim.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-scaffolder-backend': patch
```

Comment:
```suggestion
'@backstage/plugin-scaffolder-backend': minor
```

Reply:
Hi @benjdlambert ... I see you just merged. 😄 I guess you can appreciate that I was attempting in my original approach to avoid the creation of a new handler function for each action invocation, but if you feel all right with it then it's okay with me. I will say that in the current structure the `createTemplateActionHandler` function doesn't really so much _create_ anymore as just perform the action.

Reply:
yeah exactly, but appreciate the motivation to keep it clean. Thinking it's fine for now, and if we run into problems happy to revisit the refactor a little bit more.